pkg/util/docker/cgroup.go

```diff
@@ -464,6 +464,7 @@ func parseCgroupMountPoints(r io.Reader) map[string]string {
 			}
 		}
 	}
+	log.Warnf("current cgroup root is: %s", cgroupRoot)
```

Comment:
We should avoid log spam, I'd just log if we have an actual warning:
```
if len(mountPoints) == 0 {
    log.Warnf("could not locate cgroup folders in %s, please check your configuration", cgroupRoot)
}

Reply:
Sorry I immediately realized that, change already made.